lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -552,7 +553,8 @@ static bool isAssertOp(hw::HWSymbolCache &symCache, Operation *op) {
     return false;
   }
 
-  return isa<AssertOp, FinishOp, FWriteOp, AssertConcurrentOp, FatalOp>(op);
+  return isa<AssertOp, FinishOp, FWriteOp, AssertConcurrentOp, FatalOp,
+             verif::AssertOp>(op);
```

Comment:
Can't wait for layers to replace the need for this sort of thing!  And push what-goes-in-what-optional-layer to the user, instead of "fwrite is an assert" or having this error-string-prefix matching in an "SV" pass.

Reply:
I agree entirely.

The more serious issue with this PR is that this isn't just extracting the assert, but the LTL-based SVAs likely involve expressions which _really need to be extracted_. E.g., this was first reported due to a `===` showing up in the output and _not_ because of the assert.

Having advanced layer sink online would help with this, but it's not critical now that I think about it.